InvenTree/InvenTree/settings.py

```diff
@@ -723,7 +732,10 @@
         logger.info('OpenTelemetry tracing enabled')
 
         _t_resources = get_setting(
-            'INVENTREE_TRACING_RESOURCES', 'tracing.resources', {}, dict
+            'INVENTREE_TRACING_RESOURCES',
+            'tracing.resources',
+            default_value={},
```

Comment:
@SchrodingersGat you changed this to none in other places but {} here - is there a reason for that?

Reply:
@matmair no reason, I just missed it. Will issue a small patch